pkg/network/protocols/amqp/server.go

```diff
@@ -19,18 +22,67 @@ const (
 	User = "guest"
 	// Pass is the password to use for authentication
 	Pass = "guest"
+
+	// AmqpPlaintext is a flag to indicate that the server should be started with plaintext
+	AmqpPlaintext = false
+	// AmqpTLS is a flag to indicate that the server should be started with TLS
+	AmqpTLS = true
```

Comment:
nit: you're in a package called `amqp` so having `amqp.AmqpPlaintext` is redundant. Use `amqp.Plaintext` instead